agimus_controller/main_optim_traj.py

```diff
@@ -0,0 +1,67 @@
+import time
+import numpy as np
+import pinocchio as pin
+
+from wrapper_meshcat import MeshcatWrapper
+from wrapper_panda import PandaWrapper
+from ocp import OCPPandaReachingColWithMultipleCol
+from scenes import Scene
+
+### PARAMETERS
```

Comment:
Idem than previous comment:

In python one need to scope as much as possible.

please create a main function and call it afterwards in the if main guard:


```python
def main():
  my_main_code = true
  return False

if __name__ == "__main__":
  main()
```